### DISCUSSION TITLE
How to integrate/reflect the minted NFT in Smart Contract to a Marketplace

### DISCUSSION THREAD

**Discussion by saj404** [TENK-DAO/tenk#75] [Q&A]
I have successfully deployed a Smart Contract in testnet through this and uses it in a frontend website and it can mint properly. But I can't seem to find where will be the integration of the marketplace be found? For example once someone minted in the minting website, how will it show in Paras marketplace?

Did I miss a part of the code here or do I need to add it manually? Btw, I really love how you have an open source code and helps us understanding how this works! I hope this community grow bigger.

**Comment by willemneal** [comment]
Thanks! If you want to help support us, please send some royalty to `tenk.sputnik-dao.near`.

As for the market you need to have your contrract's address approved by the marketplace and then each user will have the option to add their token to it which costs a small amount to cover storage.

**Reply by saj404** [reply]
Awesome Would like to give support.

Regarding the part where each user will have the option, I did not quite get it. Which user is it referring to? Is it the one who purchased/minted the token? If so, where will they see this option?

**Reply by willemneal** [reply]
The owner of the token will have to go through a step to list the token on the exchange, which costs them a small fee to cover storage on the marketplace's contract.
